# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../inkscape/inkscape-0.46-desktop.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Make desktop-file-validate and rpmlint happy, handle compressed svgs (#245413)
and let it conform enough not to break the build in Rawhide.

(Lubomir Kundrak <lkundrak@redhat.com>)
https://bugs.launchpad.net/inkscape/+bug/190420

diff -up inkscape-0.46/inkscape.desktop.in.desktop inkscape-0.46/inkscape.desktop.in
--- inkscape-0.46/inkscape.desktop.in.desktop	2008-03-11 05:21:19.000000000 +0100
+++ inkscape-0.46/inkscape.desktop.in	2008-08-13 23:29:56.000000000 +0200
@@ -2,18 +2,16 @@
 _Name=Inkscape Vector Graphics Editor
 _Comment=Create and edit Scalable Vector Graphics images
 
-Encoding=UTF-8
 Version=1.0
 
 Type=Application
-Categories=Application;Graphics;VectorGraphics;GTK;
+Categories=Graphics;VectorGraphics;GTK;
 
-MimeType=image/svg+xml
+MimeType=image/svg+xml;image/svg+xml-compressed;
 
-FilePattern=inkscape
 Exec=inkscape %F
 TryExec=inkscape
 Terminal=false
 StartupNotify=true
 
-Icon=inkscape.png
+Icon=inkscape
